packages/embark-blockchain-connector/src/provider.js

```diff
@@ -22,6 +22,9 @@ class Provider {
       const accounts = this.accounts.map(a => a.address || a);
       cb(null, accounts);
     });
+    this.events.setCommandHandler("blockchain:provider:contract:accounts:getAll", (cb) => {
+      cb(null, this.accounts);
+    });
```

Comment:
Not necessarily needed, but I like to use as little characters for expressions as possible (assuming readability doesn't suffer):

```
this.events.setCommandHandler('blockchain:provider:contract:accounts:getAll', cb => cb(null, this.accounts))
```

Either way, your call :)

Reply:
The reason for choosing `"blockchain:provider:contract:accounts:getAll"` is because the existing command `"blockchain:provider:contract:accounts:get"` and I guess was an attempt at keeping consistency since they are very similar commands.

Reply:
Oh the event name is fine, I was more referring to the code itself (I proposed a one-liner).

Anyways :)